### PR TITLE
worker id info added to Running state

### PR DESCRIPTION
**Pull request by fredykoci**

### CODE REVIEW

File: src/messages/gateway.rs [hunk 1]
```diff
@@ -95,7 +97,7 @@ impl Serialize for TaskState {
             TaskState::Invalid => "Invalid",
             TaskState::Waiting => "Waiting",
             TaskState::Finished => "Finished",
-            TaskState::Running => "Running",
+            TaskState::Running(_) => "Running",
```
Comment (vyomkeshj) [inline src/messages/gateway.rs:100]: @Kobzol  I wonder if there are implications that the serialization is missing the information about the worker_id

Comment (Kobzol) [reply src/messages/gateway.rs:100]: In theory, we might need to change the serialization from a string to an enum, but worker placement might be much more complex than a single integer, so maybe this information should be propagated in another way altogether.

Comment (Kobzol) [inline src/messages/gateway.rs:100]: The serialization here needs to be changed. It's a more complex change, so we will probably do it in another PR. @spirali

Comment (fredykoci) [reply src/messages/gateway.rs:100]: there can not be serialization changed everythink works well
